neutron_vpnaas/services/vpn/device_drivers/ovn_ipsec.py

```diff
@@ -264,9 +283,20 @@ def get_router_based_iptables_manager(self, router):
 
     def destroy_process(self, process_id):
         LOG.info(_LI('process %s is destroyed') % process_id)
+        namespace = self.devmgr.get_namespace_name(process_id)
+
+        if self.conf.meter.vpn_meter_enable:
```

Comment:
can we wrap all these logical into one function? That would be must simple for reference. And as a meter library, it is better to export less APIs.